PubnativeLite/PubnativeLite/Location/PNLiteLocationManager.m

```diff
@@ -98,6 +98,7 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
 
 - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations {
     self.lastKnownLocation = locations.lastObject;
+    [[PNLiteLocationManager sharedInstance].manager stopUpdatingLocation];
 }
```

Comment:
All good! it should be stopped directly after receiving first location update. 
In this case, we make sure that we always gets ONE accurate location and don't drain the phone battery by multiple requests.

Reply:
okay then... so the place for stoping it also makes sense i guess right @faresbh?  it is right after we set a location update. and it also guarantees so stop location update every time we request one

Reply:
yes, it's the perfect place for our case.